backend/tournesol/migrations/0025_create_default_criterias.py

```diff
@@ -0,0 +1,95 @@
+# Generated by Django 4.0.2 on 2022-02-08 15:26
+
+from email.policy import default
+from django.db import migrations
+
+CRITERIAS = [
+    {
+        "name": "largely_recommended",
+        "label_en": "Should be largely recommended",
+        "label_fr": "Devrait être largement recommandé",
+        "optional": False,
+    },
+    {
+        "name": "reliability",
+        "label_en": "Reliable & not misleading",
+        "label_fr": "Fiable & non trompeur",
+    },
+    {
+        "name": "pedagogy",
+        "label_en": "Clear & pedagogical",
+        "label_fr": "Clair & pédagogique",
+    },
+    {
+        "name": "importance",
+        "label_en": "Important & actionable",
+        "label_fr": "Important & actionnable",
+    },
+    {
+        "name": "layman_friendly",
+        "label_en": "Layman-friendly",
+        "label_fr": "Accessible aux non-spécialistes",
+    },
+    {
+        "name": "entertaining_relaxing",
+        "label_en": "Entertaining & relaxing",
+        "label_fr": "Divertissant & relaxant",
+    },
+    {
+        "name": "engaging",
+        "label_en": "Engaging & thought-provoking",
+        "label_fr": "Stimulant & suscite la réflexion",
+    },
+    {
+        "name": "diversity_inclusion",
+        "label_en": "Diversity & inclusion",
+        "label_fr": "Diversité & inclusion",
+    },
+    {
+        "name": "better_habits",
+        "label_en": "Encourages better habits",
+        "label_fr": "Encourage de meilleures habitudes",
+    },
+    {
+        "name": "backfire_risk",
+        "label_en": "Resilience to backfiring risks",
+        "label_fr": "Résistant aux retours négatifs",
+    },
+]
+
+
+def migrate_forward(apps, schema_editor):
+    """
+    Set the `uid` and the `type` field of all entities present in the
+    database (supposedly only YT videos are present).
+    """
```

Comment:
This seems to be and old comment : )

```suggestion
    """
    Create the criteria of the `videos` poll.
    """
```